pkg/common/patterns.go

```diff
@@ -58,14 +58,14 @@ func (r RegexState) Matches(data []byte) []string {
 
 // UsernameRegexCheck constructs an username usernameRegex pattern from a given pattern of excluded characters.
 func UsernameRegexCheck(pattern string) RegexState {
-	raw := fmt.Sprintf(`(?im)(?:user|usr)\S{0,40}?[:=\s]{1,3}[ '"=]{0,1}([^:%+v]{4,40})\b`, pattern)
+	raw := fmt.Sprintf(`(?im)(?:user|usr)\S{0,40}?[:=\s]{1,3}[ '"=]{0,1}([^:%+v'"\s]{4,40})`, pattern)
```

Comment:
I'm pretty uncomfortable changing these "common" patterns in a PR that also adds a new detector, because one change is very general and one is very targeted. That suggests they should be in different PRs.

_However_, these "common" patterns are only used in three places. I'm not sure why they were made common in the first place - couchbase, snowflake, and postgres don't actually have any inherent relationship. This reads like it was purely mechanical code deduplication, which isn't usually a good idea. This, in turn, suggests re-duplicating the patterns into the specific locations they're used and just changing the ones that need changing. In this PR, that would just mean putting these patterns directly into the postgres detector and leaving these "general" changes to a future PR.

Reply:
done.